docs/ref_lifecycle.md

```diff
@@ -0,0 +1,18 @@
+[◀︎ Airlift](../README.md) • [◀︎ Reference](../README.md#reference)
+
+## Reference: Lifecycle/Bootstrapping
+
+### @PostConstruct and @PreDestroy
+
+Airlift enables the standard [@PostConstruct](https://docs.oracle.com/javase/8/docs/api/javax/annotation/PostConstruct.html) and [@PreDestroy](https://docs.oracle.com/javase/8/docs/api/javax/annotation/PreDestroy.html) annotations for objects injected/bound
```

Comment:
Do we want to use code quotes inside links
```markdown
[`@PostConstruct`](https://docs.oracle.com/javase/8/docs/api/javax/annotation/PostConstruct.html)

Reply:
I tried that and the formatting was odd